renku/core/commands/checks/external.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Checks for external files."""
+from pathlib import Path
+
+import click
+
+from renku.core.commands.echo import WARNING
+
+
+def check_missing_external_files(client):
+    """Find external files that are missing."""
+    missing = []
+
+    for path, dataset in client.datasets.items():
+        for file_ in dataset.files:
+            if file_.external:
+                target = Path(file_.path).resolve()
+                if not target.exists():
+                    missing.append((file_.path, str(target)))
+
+    if not missing:
+        return True, None
+
+    problems = (
+        '\n' + WARNING + 'There are missing external files.\n'
+        '  (make sure that external volume is mounted and accessible)' +
```

Comment:
This works really nicely! Just a minor wording suggestion - in most cases it might not be a mounted volume so the user might wonder what that's all about. 

```suggestion
        '  (make sure that the external path is accessible)' +
```